CMakeLists.txt

```diff
@@ -1,5 +1,25 @@
 cmake_minimum_required(VERSION 3.20)  # For using CMAKE_<LANG>_BYTE_ORDER
 
+if(NOT DEFINED USE_VCPKG OR USE_VCPKG)
+  message("NOT DEFINED USE_VCPKG OR USE_VCPKG")
```

Comment:
These look like internal debug messages. User output should be `Using system-provided libraries` or `Using VCPKG` if you want to keep some feedback

Reply:
🤦‍♂️